workers/unity/Packages/com.improbable.gdk.gameobjectcreation/GameObjectInitializationSystem.cs

```diff
@@ -12,38 +12,39 @@ namespace Improbable.Gdk.GameObjectCreation
     ///     and links it to the entity via the EntityGameObjectLinker. Also checks for entity removal and calls the
     ///     IEntityGameObjectCreator for cleanup.
     /// </summary>
-    [UpdateInGroup(typeof(SpatialOSReceiveGroup.EntityInitialisationGroup))]
     [DisableAutoCreation]
     internal class GameObjectInitializationSystem : ComponentSystem
     {
-        // TODO: Add our ISystemStateComponentData for all entities instead of ones with GameObjects, store spatial ID
-        // callbacks instead of in it, give that in removal callback instead of SpatialOSEntity struct.
+        public struct HasGameObjectSystemState : ISystemStateComponentData
+        {
+            public SpatialEntityId EntityId;
+        }
 
         private struct AddedEntitiesData
         {
             public readonly int Length;
-            public EntityArray Entities;
+            [ReadOnly] public EntityArray Entities;
             [ReadOnly] public ComponentDataArray<SpatialEntityId> SpatialEntityIds;
-            [ReadOnly] public ComponentDataArray<NewlyAddedSpatialOSEntity> NewlyCreatedEntities;
+            [ReadOnly] public SubtractiveComponent<HasGameObjectSystemState> NoHasGameObjectSystemStates;
```

Comment:
Minor: `NoHas` reads kind of strange, perhaps in this case `HasNo` or `DoesNotHave` or something may work better.

Reply:
`DenotesThereIsNoGameObject`

Reply:
`Denotes`also fits with how we are doing it elsewhere, so that sounds and reads nicely. Not adding an underscore though :P